src/Reflex/Dom/Pandoc/Document.hs

```diff
@@ -224,6 +224,7 @@ renderInline cfg = \case
     elPandocAttr "span" attr $
       renderInlines cfg xs
   where
+    title t = memptyIfTrue (T.null t) ("title" =: t)
```

Comment:
You can get rid of this, now that we are using Map.filter ...

Reply:
I tried getting rid of it. It doesn't work because the map filter only applies to the `Attr`

```haskell
--                 v-- Not filtered
(renderAttr attr) <> ("src" =: iUrl <> title iTitle)
--         ^-- Map.filter works here
```

Reply:
Shouldn't this work (note that `$`):

```haskell
renderAttr $ attr <> "src" =: iUrl <> title iTitle
```

Reply:
Okay, I think we should refactor this into two functions. 

* `pandocAttrToReflexAttrs` (renamed from `renderAttr` for clarity) -> converts the rather odd type (the pandoc 3-tuple) into a reflex attribute Map
* `sansEmptyAttrs` -> transforms that Map to discard empty attr values

Then you can do,

```haskell 
let attr' = 
    sansEmptyAttrs $ 
           pandocAttrToReflexAttrs attr 
        <> "src" =: iUrl 
        <> "title" =: iTitle
```

Reply:
Okay, I think we should just add a simple `sansEmptyAttrs` function that transforms that Map to discard empty attr values

Then you can do,

```haskell 
let attr' = 
      sansEmptyAttrs $ 
           renderAttr attr 
        <> "src" =: iUrl 
        <> "title" =: iTitle
```

Reply:
(And of course remove the Map.filter from renderAttr)